src/meshes/p4est_icosahedron_quads.jl

```diff
@@ -0,0 +1,774 @@
+@muladd begin
+#! format: noindent
+
+# Using element_local mapping
+"""
+    P4estMeshCubedSphere2D(trees_per_face_dimension, radius;
+                            polydeg, RealT=Float64,
+                            initial_refinement_level=0, unsaved_changes=true,
+                            p4est_partition_allow_for_coarsening=true)
+
+Build a quad-based icosahedral mesh as a 2D `P4estMesh` with
+`60 * trees_per_face_dimension^2` trees (20 triangular faces of the icosahedron,
+each subdivided into 3 parent quads, each of which subdivided into `trees_per_face_dimension^2` trees).
+
+The node coordinates of the trees will be obtained using the element-local mapping from
+Appendix A of [Guba et al. (2014)](https://doi.org/10.5194/gmd-7-2803-2014). The four corner vertex 
```

Comment:
I would remove everything from "The four corner vertex positions..." onward, and perhaps just add a reference to `P4estMeshCubedSphere2D` regarding the mapping from Guba et al.

Reply:
Done in cd6bffab3dc57b4232c06e12549507702386a8d5.